### PR TITLE
*: cut 2.2.0-rc.1

### PR DESCRIPTION
**Pull request by fabxc**
@brian-brazil

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 
 * [CHANGE] Rename file SD mtime metric.
+* [CHANGE] Send target up on empty pod IP in Kubernetes SD.
 * [ENHANCEMENT] Federation performance improvement.
 * [ENHANCEMENT] Read bearer token file on every scrape.
 * [ENHANCEMENT] Improve typeahead on `/graph` page.
 * [ENHANCEMENT] Change rule file formatting.
 * [ENHANCEMENT] Set consul server default to `localhost:8500`.
+* [ENHANCEMENT] Add dropped Alertmanagers to API info endpoint.
+* [ENHANCEMENT] Add API endpoint for flags.
```
Comment (brian-brazil) [inline CHANGELOG.md]: FEATURE

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+* [ENHANCEMENT] Add OS type meta label to Azure SD.
+* [BUGFIX] Prevent stack overflow on deep recursion in TSDB.
+* [BUGFIX] Correctly read offsets in index files that are greater than 4GB.
+* [BUGFIX] Fix honor_labels behavior for empty labels.
```
Comment (brian-brazil) [inline CHANGELOG.md]: And empty labels behaviour generally

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,18 @@
-## 2.2.0-rc.0 / 2018-02-13
+## 2.2.0-rc.1 / 2018-02-21
 
 * [CHANGE] Rename file SD mtime metric.
+* [CHANGE] Send target up on empty pod IP in Kubernetes SD.
```
Comment (brian-brazil) [inline CHANGELOG.md]: s/up/update/

Comment (brian-brazil) [reply CHANGELOG.md]: This also sounds more like a bugfix

**Comment by fabxc** [comment]
done

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**